cluster/src/main/scala/io/snappydata/tools/SnappyUtilLauncher.scala

```diff
@@ -171,9 +178,82 @@ object SnappyUtilLauncher extends StoreCallback {
     }
   }
 
+  val intpUsage = s"\nUsage:\n" +
+    s"\n${SCRIPT_NAME} [--snappydata.connection | -conn | --connection locatorhost:port]" +
+    s"\n             default = locahost:1527\n" +
+    s"\n             [--snappydata.user | -user | --user username]" +
+    s"\n             default = APP\n" +
+    s"\n             [--snappydata.password | -passwd | --password password]" +
+    s"\n             default = APP\n" +
+    s"\n             [--run | -r | -run scala_file path or files (comma separated paths]" +
+    s"\n             [--help | -help | -h 'prints the command line options for the script']\n"
+
+
+  private def validateArgs(options: Map[Symbol, Any]): Unit = {
+    val hostport = options.get('connection) match {
+      case Some(x) => x.toString
+      case None => "localhost:1527"
+    }
+    val user = options.getOrElse('user, "APP")
+    val passwd = options.getOrElse('password, "APP")
+    CONNECT_STR = s"connect client '${hostport};user=${user};password=${passwd}'"
+    val filesToRun = options.get('run) match {
+      case Some(x) => val files = x.toString
+        files.split(",").foreach(p => {
+          val path = Paths.get(p)
+          if (!Files.exists(path)) {
+            println(s"File $path does not exists")
+            System.exit(1)
+          }
+        })
+        files
+      case None => ""
+    }
+    if (filesToRun.nonEmpty) {
+      INITIAL_FILES_TO_RUN_STR = filesToRun.toString
+    }
+  }
+
+  // expected comma separated files
+  private var INITIAL_FILES_TO_RUN_STR: String = null
+  private var CONNECT_STR: String = null;
+
+  private def validateIntpArgs(args: Array[String]): Unit = {
+    val arglist = args.toList
+    type ArgsMap = Map[Symbol, Any]
+
+    def isSwitch(s : String) = (s(0) == '-')
+
+    def nextOption(map : ArgsMap, list: List[String]) : ArgsMap = {
+
+      list match {
+        case Nil => map
+        case ("--snappydata.connection" | "-conn" | "--connection" ) :: value :: tail =>
```

Comment:
- Make sure this is included in the Spec... and, hence in the docs. 
- why are there so many ways to connect ? shouldn't we stick to the convention of --conf spark.snappydata.connection or --conf snappydata.connection  only ?

Reply:
cleaned up the help. Avoiding --conf thingy as users might get confused that they can pass in spark properties as well, which is not the case.